demosplan/DemosPlanCoreBundle/Resources/client/scss/token/index.scss

```diff
@@ -8,6 +8,6 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // Each file defines token variables.
-@import 'box-shadow';
+@import '@demos-europe/demosplan-ui/tokens/scss/_boxShadow';
```

Comment:
I'd suggest to import those tokens directly in `DemosPlanCoreBundle/Resources/client/scss/core_style.scss:44`, so we can drop the `/scss/token/` folder entirely in the end.

Another point: when importing the scss partial, you don't need the underscore:

```scss
@import '@demos-europe/demosplan-ui/tokens/scss/boxShadow';
```

should work as well.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1307/commits/a8203013c01b2cd5cd370001f81d7830b1187fdb, thanx